src/helm/benchmark/scenarios/bhasa_scenario.py

```diff
@@ -171,7 +171,7 @@ def __init__(self, language: str):
         super().__init__()
         self.language = language
         self.splits = {"train": TRAIN_SPLIT, "test": TEST_SPLIT}
-        self.map = {
+        self.prompts = {
```

Comment:
Rename to `self.language_to_prompt_components`.

Same below.

Reply:
Moved all variable components to `self.language_to_prompt_components`.